_about/notes/index.md

```diff
@@ -1,7 +1,5 @@
 ---
 title: Release Notes
-overview: What's been happening with Istio.
```

Comment:
Why remove the overview? That string is used for the hover text in the sidenav, and as the description test if you go to about/index.html

Reply:
the text showed up on the page and looked strange with the new content - do you have a proposed new/better text or you want me to keep this one ?

Reply:
updated. ptal